guidelines/content-guidelines/release-notes.md

```diff
@@ -0,0 +1,29 @@
+# Release notes
+Information in release notes must provide readers with everything they need to know to understand the change in the software. A lot of business decisions are made based on the information in release notes. Therefore, always write from the user's perspective, not the developer's perspective. The content of release notes answers the following questions:
+
+* What has changed because of this feature or resolved issue?
+* How was the behavior different before this release?
+* Are there changes to the UI?
+* Are there changes to the functionality?
+* Does an error message appear?
+* Was the enhancement based upon customer feedback?
+
+## Guidelines
+Because the release notes contain critical information and act as an important communication tool, follow these guidelines so that the documentation is informative and consistent. When authoring release notes, follow the [Style and Standards](./style-and-standards.md) for many agreed-upon standards.
+
+## Headlines
```

Comment:
How about making this and the following sections H3? All of them fall into `## Guidelines`, don't they?